src/components/Global/DropdownMenu2/DropdownMenu2.styles.ts

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import { motion } from 'framer-motion';
+
+export const Menu = styled.div`
```

Comment:
for all of these styled components that contains `display: flex`, can we extend the FlexContainer class as discussed?

Reply:
done